### PR TITLE
[Meru] New naming screen

### LINKED ISSUE #3785: [Meru - AB] Layout - New Naming screen
Details on what to do in the **[Figma](https://www.figma.com/file/QOxHwppG64GtPocpDhS6Y7/Product?type=design&node-id=4900-39387&mode=design&t=6tNvt0C7wUTsOHc4-4)** (also, see how it works in the [Prototype](https://www.figma.com/proto/QOxHwppG64GtPocpDhS6Y7/Product?page-id=3366%3A31345&type=design&node-id=4900-40669&viewport=1173%2C2546%2C0.27&t=lV2M2VcNHLWO3Bpe-1&scaling=min-zoom&starting-point-node-id=4900%3A40669) + context & discussion in the [making issue](https://github.com/dust-tt/decisions/issues/164))

- "Suggestions" are in a separate issue, out of scope for this one
- Emojis & Dropzone for avatar are also in a separate issue - reuse existing modal for this one BUT moved in a drawer 
- Pretty straightforward otherwise, goal is to match the figma closely

### PR DESCRIPTION
**Pull request by philipperolet**
## Description
Fixes #3785 

Please see follow-up coms for discussable options
image
image

### CODE REVIEW

**Comment by philipperolet** [comment]
### Error message cc @Duncid 
1. I put the triangle warning symbol. I can use the circled info symbol if needed. The circled exclamation mark is not straightforward (less common, might be compatibility issues etc.) so unless really important 
2. Color of the message => Sparkle says red (Input component), figma says grey. You told me sparkle wins against figma, but just wanted to check
3. another wording suggestion vs the figma "This handle is already taken"

**Comment by philipperolet** [comment]
### Wording "Handle"

I would not use a complex term when a simple one ('name') does 95% of the job. I get there is a redundancy with the "Naming", but "handle" is a little raise on the learning curve to use dust (new vocabulary, painful, technical, geeky) with a limited benefit. Think of Claudine de la compta.

**Comment by philipperolet** [comment]
### Avatar: no title in modal from figma?
(fine by me, just checking)

File: front/components/assistant_builder/AssistantBuilder.tsx [hunk 1]
```diff
@@ -438,7 +420,7 @@ export default function AssistantBuilder({
         }
         valid = false;
       } else if (!(await assistantHandleIsAvailable(builderState.handle))) {
-        setAssistantHandleError("Assistant handle is already taken");
+        setAssistantHandleError("\u26a0 This handle is already taken");
```
Comment (fontanierh) [inline front/components/assistant_builder/AssistantBuilder.tsx:423]: Do we really want the unicode char vs an icon ?

**Review by fontanierh — APPROVED** [review]
code LGTM 👍

File: front/components/assistant_builder/AssistantBuilder.tsx (hunk 1 above)
Comment (philipperolet) [reply front/components/assistant_builder/AssistantBuilder.tsx:423]: In this specific case I don't think there is a benefit to use an icon. Are unicode chars an issue? (this one is standard)

**Pull request merged**